tests/test_thumbnail.py

```
@@ -31,0 +1,250 @@
+            result = thumbnail._create_thumbnail_html_playwright("dummy.html", "out.jpg", self.test_width, self.test_height, self.test_format, self.test_quality)
+
+        self.assertTrue(result)
+        mock_page.goto.assert_called_once() # Argument can be checked if it's more complex
+        mock_page.screenshot.assert_called_once_with(full_page=False)
+        mock_pil_image_open.assert_called_once() 
+        self.mock_pil_image.thumbnail.assert_called_once_with((self.test_width, self.test_height))
+        mock_makedirs.assert_called_once_with(os.path.dirname("out.jpg"), exist_ok=True)
+        self.mock_pil_image.save.assert_called_once_with("out.jpg", self.test_format, quality=self.test_quality)
+        mock_browser.close.assert_called_once()
+
+    @patch('thumbnail.PIL_AVAILABLE', False)
```

Ruff (0.11.9)

Boolean positional value in function call

(FBT003)